manifests/hourly.pp

```diff
@@ -16,19 +16,27 @@
   Enum['present','absent'] $ensure = 'present'
 ) {
 
+  $manage_cron_hourly = $::logrotate::manage_cron_hourly
```

Comment:
please do not add the leading ```::```, those are only needed for puppet 3 and are now legacy.

Reply:
Can you point me at the documentation that states these are legacy as I use them extensively throughout my profile and other component modules?

Reply:
We have the following in our [review guidelines](https://voxpupuli.org/docs/#reviewing-a-module-pr):
> * In the majority of cases, variables shouldn’t be accessed via topscope: $::modulename::$param. Instead do: $modulename::$param

This is also somehwere documented within the official docs, but I do not have the link by hand.

Reply:
Thanks